src/action/wallet.js

```diff
@@ -265,6 +266,7 @@ class WalletAction {
    */
   async initWallet({ walletPassword, seedMnemonic, recoveryWindow = 0 }) {
     try {
+      await Promise.all([this.deleteDB('testnet'), this.deleteDB('mainnet')]);
```

Comment:
Just a note: if you're going through this PR's reset flow, but you use the wrong seed, wallet initialization will fail every time. `Wallet initialization failed: undefined`

Reply:
@Roasbeef any ideas?

Reply:
> Just a note: if you're going through this PR's reset flow, but you use the wrong seed, wallet initialization will fail every time. `Wallet initialization failed: undefined`

@valentinewallace I added a try/catch case and specific error message for this case. It's in the current testnet build.